content/api/vision.md

```diff
@@ -5,14 +5,14 @@ title: Vision
 Your webcam is how your robot will see markers.
 
 ## Using the camera to search for markers
-If you have a webcam connected, you can use `r.camera.see()` to take a picture. 
+If you have a webcam connected, you can use `r.camera.see()` to take a picture.
 
 If you consider it necessary you can assign the camera object to a variable just like `Robot`:
```

Comment:
This sentence is amazingly misleading (in terms of what `Robot` is as compared to `r.camera`). Can we just drop it and the following code snippet? (maybe a separate PR)

Reply:
I've removed the unnecessary reference to `Robot` in 14e4544802301cb35f6df2c833908349bbbe6635, but left the rest as is.

Separately, I feel this block (and the corresponding ones for the power, motor and servo boards) don't add anything to the docs as they are. Users who are competent enough to structure their code in a way that lends itself to only passing around a single board rather than the entire `Robot` instance probably grasp the concept of attribute access already. For those who don't this snippet may just add to the confusion, as we don't clearly explain what the _purpose_ of doing this is or in what circumstances it's _necessary_.

I'd recommend we either try and make them much more useful, or just remove them on the basis of them being confusing. Either way this would be a separate PR.

Reply:
I _think_ it was introduced as we saw some SR competitors doing approximately the following (which doesn't work):
``` python
m0 = R.motors[0].m0
m0 = 50 # Set to 50% power
```

Since SourceBots' API has the potential for similar downfalls, I can see why theses are here; however I agree that they don't really feel like they add a lot in their current form.